cmd/maya-apiserver/cstor-operator/spc/handler.go

```diff
@@ -142,6 +175,57 @@ func (c *Controller) create(pendingPoolCount int, spc *apis.StoragePoolClaim) er
 	return nil
 }
 
+// cleanUPSPCChild will remove pool related resources and finalizers on BDC
+func (c *Controller) cleanUpSPCChild(spc *apis.StoragePoolClaim) error {
```

Comment:
`cleanUpSPCChild` ---> `cleanupSPCChild` as cleanup is a single word.

Reply:
I Will do

Reply:
instead `handleSPCDeleteEvent` or the name similar to what done for CSPC